<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page2_login</class>
 <widget class="QMainWindow" name="page2_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>541</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="username_lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>User Name here...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Password_lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Password here...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Verification_Password_lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Verification Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signin_pushButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>360</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>sign in</string>
     </property>
    </widget>
    <widget class="QLabel" name="Question_label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>320</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Do not have account?</string>
     </property>
    </widget>
    <widget class="QLabel" name="capcha_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="capcha_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>capcha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="capcha_lineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="next_pushButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>320</y>
       <width>155</width>
       <height>54</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="showhide_pushButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>40</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="symbol_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>symbol</string>
     </property>
    </widget>
    <widget class="QLabel" name="character_label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>90</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>7 character</string>
     </property>
    </widget>
    <widget class="QLabel" name="verification_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>verification password</string>
     </property>
    </widget>
    <widget class="QLabel" name="check_username_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>username has been exist</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>-80</y>
       <width>800</width>
       <height>600</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>username_lineEdit</zorder>
    <zorder>Password_lineEdit</zorder>
    <zorder>Verification_Password_lineEdit</zorder>
    <zorder>signin_pushButton</zorder>
    <zorder>Question_label</zorder>
    <zorder>capcha_label</zorder>
    <zorder>capcha_pushButton</zorder>
    <zorder>capcha_lineEdit</zorder>
    <zorder>next_pushButton</zorder>
    <zorder>showhide_pushButton</zorder>
    <zorder>symbol_label</zorder>
    <zorder>character_label</zorder>
    <zorder>verification_label</zorder>
    <zorder>check_username_label</zorder>
   </widget>
   <widget class="QPushButton" name="back_pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
